src/strategies/orange-reputation-based-voting/index.ts

```diff
@@ -0,0 +1,44 @@
+import fetch from 'cross-fetch';
+
+export const author = 'orange-protocol';
+export const version = '0.1.0';
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  const query = `{
+    getBasedVotingStrategy(
+      addrs: ${JSON.stringify(addresses)},
+      space: "${space}",
+      snapshot: "${snapshot}",
+      network: "${network}",
+      provider: "${provider}",
```

Comment:
Provider is not a string or number, I assume it wont work well here, you'd rather not having it at all

Reply:
@bonustrack This is input for the oraganprotocol API https://api.orangeprotocol.io/orange2c/query
I can't find their documentation or schema 🤔

Reply:
But this return the variable from the strategy with Ethers.js provider https://github.com/snapshot-labs/snapshot-strategies/pull/254/files#diff-b60e2ce70275e1efd45ba615c092306ebd2de4585a540f539a16f1ba91ed4ed0R9